modules/assignment/src/main/resources/assignment.properties

```diff
@@ -25,4 +25,15 @@ decision.engine.use.connection.pooling:${decision.engine.use.connection.pooling}
 decision.engine.max.connections.per.host:${decision.engine.max.connections.per.host}
 http.proxy.host:${http.proxy.host}
 http.proxy.port:${http.proxy.port}
-ruleCache.executor.pool.size:5
\ No newline at end of file
+ruleCache.executor.pool.size:5
+
+#Flag to enable or disable assignments metadata cache.
+metadata.cache.enabled:true
+
+#This is a cache refresh interval, at this frequency cache will be refreshed.
+metadata.cache.refresh.interval:5
```

Comment:
@bargenilesh how many minutes for 1 interval? will be helpful to include that in the comment.